docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Docker build file to create a centos image for building rpm packages
+
+FROM centos:7
+MAINTAINER andreas-bok-sociomantic <andreas-bok@sociomantic.com>
```

Comment:
Isn't it `andreas.bok` ? Maybe we could have a public address for the project(s) ?